apps/server/src/Scoring.ts

```diff
@@ -0,0 +1,60 @@
+let correctSequence = ['A', 'A', 'B', 'C', 'D'];
+let noteIndex = 0;
+let combo = 0;
+
+function setNoteSequence(seq: string[]) {
+  correctSequence = seq;
+  noteIndex = 0;
+  UpdateCombo(false);
+}
```

Comment:
I think we should add return infer type in this function as well to keep the code consistency